spec/setup_test_database.rb

```diff
@@ -0,0 +1,10 @@
+require 'pg'
+
+def setup_test_database
+  p "Setting up test database..."
+
+  connection = PG.connect(dbname: 'peep_manager_test')
+
+  # Clear the bookmarks table
+  connection.exec("TRUNCATE peeps;")
+end
```

Comment:
good idea to move code into a helper file, even if it wasn't used in this most recent commit